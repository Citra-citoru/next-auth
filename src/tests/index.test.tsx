import { render, screen } from '@testing-library/react'
import Home from '@/app/page'

describe('Home', () => {
  it('renders a heading', () => {
    const { container } = render(<Home />)

    const heading = screen.getByRole('heading', {
      name: /Home Page!/i,
    })

    expect(heading).toBeDefined()

    expect(container).toMatchSnapshot()
  })
})
