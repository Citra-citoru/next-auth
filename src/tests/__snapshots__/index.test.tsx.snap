// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders a heading 1`] = `
<div>
  <h1>
    Home Page!
  </h1>
  <a
    href="/login"
  >
    Login
  </a>
  <a
    href="/signup"
  >
    Signup
  </a>
</div>
`;
